packages/interactivity/src/proxies/context.ts

```diff
@@ -38,6 +38,9 @@ const contextHandlers: ProxyHandler< object > = {
 	getOwnPropertyDescriptor: ( target, key ) =>
 		descriptor( target, key ) ||
 		descriptor( contextObjectToFallback.get( target ), key ),
+	has: ( target, key ) =>
+		Reflect.has( target, key ) ||
+		Reflect.has( contextObjectToFallback.get( target ), key ),
```

Comment:
> those lines are on the unproxied objects so that usage of `in` is not impacted. Is that correct?

Correct. 🙂



> Are there tests already for the case where the value should be set on fallback?

Yes, there are. We have a couple of unit tests checking prop modifications on fallback contexts:

https://github.com/WordPress/gutenberg/blob/67b7e0450e3569346c276fc2b8ed664bf485127c/packages/interactivity/src/proxies/test/context-proxy.ts#L77-L158